tests/appsec/iast/sink/test_unvalidated_redirect.py

```diff
@@ -21,15 +43,23 @@ class TestUnvalidatedRedirect:
         insecure_endpoint="/iast/unvalidated_redirect/test_insecure_header",
         secure_endpoint="/iast/unvalidated_redirect/test_secure_header",
         data={"location": "http://dummy.location.com"},
-        location_map={"java": {"spring-boot": "com.datadoghq.system_tests.springboot.AppSecIast"}},
+        location_map=_expected_location,
     )
     sink_fixture_redirect = SinkFixture(
         vulnerability_type="UNVALIDATED_REDIRECT",
         http_method="POST",
         insecure_endpoint="/iast/unvalidated_redirect/test_insecure_redirect",
         secure_endpoint="/iast/unvalidated_redirect/test_secure_redirect",
         data={"location": "http://dummy.location.com"},
-        location_map={"java": {"spring-boot": "com.datadoghq.system_tests.springboot.AppSecIast"}},
+        location_map=_expected_location,
+    )
+    sink_fixture_forward = SinkFixture(
```

Comment:
We could just have a different file and class for `test_unvalidated_forward.py`, I'm fairly sure that the `@release` decorator and implementation timeframes will generally be quite different for both across languages.